docs/GlobalAPI.md

```diff
@@ -0,0 +1,332 @@
+---
+id: api
+title: Globals
+layout: docs
+category: API Reference
+permalink: docs/api.html
+next: expect
+---
+
+In your test files, Jest puts each of these methods and objects into the global environment. You don't have to require or import anything to use them.
```

Comment:
Actually, `jest` is not a global. It's a module local object similar to `module` and `require` and it is different for each file.